eng/deploy-sql.ps1

```diff
@@ -73,7 +76,7 @@ Get-ChildItem "$PSScriptRoot/../sql/mssql/*.sql" | `
 
             if ($result.Count -gt 0) {
                 # Script has already run, so skip it
-                continue
+                return
```

Comment:
Something a little different in PowerShell compared to C# and other languages: the `continue` acts like a `break` statement in other languages! But the `return` inside a loop acts like other languages' `continue`.